app/routes/projects/$projectId/index.tsx

```diff
@@ -185,13 +192,22 @@ export default function ProjectDetailsPage() {
             </Grid>
             <Grid item>
               {(isTeamMember || isAdmin) && (
-                <IconButton
-                  aria-label="Edit"
-                  href={`/projects/${project.id}/edit`}
-                >
-                  <EditSharp />
-                </IconButton>
+                <Tooltip title="Edit project">
+                  <IconButton
+                    aria-label="Edit"
+                    href={`/projects/${project.id}/edit`}
+                  >
+                    <EditSharp />
+                  </IconButton>
+                </Tooltip>
               )}
+
+              {(isTeamMember || isAdmin) &&
```

Comment:
Since you are doing the same validation as in the edit button, I think you don't need to create your own boolean block, but just add the IconButton to the top one.

Also, I think it would be better if the edit button is the one on the rigth.